alarm/ExceptionManager.js

```diff
@@ -487,4 +487,40 @@ module.exports = class {
 
     return _.uniqWith(matchedExceptions.map((exception) => exception.eid), _.isEqual);
   }
+
+  async batchException(actions) {
+    let results = {
+      'create': [],
+      'update': [],
+      'delete': []
+    };
+    for (const action in actions) {
+      const rawData = actions[action] || [];
+      switch (action) {
+        case 'create':
+        case 'update':
+          const emAction = action == 'create' ? 'createException' : 'updateException';
```

Comment:
this could break some IDE functionalities, maybe
```
const emAction = action == 'create' ? this.createException : this.updateException
emAction.bind(this)(exception)
```